api/services/authservice.js

```diff
@@ -0,0 +1,64 @@
+const crypto = require('crypto');
+const pool = require('../db/queries');
+const jwt = require('jsonwebtoken');
+
+class AuthService {
+    constructor() {
+        this.accessTokenSecret = 'jdhhhhseddassbckdbsdkbehoa';
+    }
+
+    async SignUp(name, email, password) {
+        const passwordHashed = this.hashPassword(password);
+
+        await pool.query('SELECT * FROM users WHERE email = $1', [email])
+            .then(result =>{ 
+                if (result.rows.length !== 0) {
```

Comment:
move bool calculations to separate const 
live 
const isResultNotEmpty = result.rows.length !== 0
if(isResultNotEpmty)